src/web-test-runner.config.mjs

```diff
@@ -0,0 +1,11 @@
+import { chromeLauncher } from '@web/test-runner-chrome';
+import { defaultReporter } from '@web/test-runner';
+
+export default (/** @type {import('@web/test-runner').TestRunnerConfig} */{
+  rootDir: './',
+  files: './build/scripts/tests.bundle.js',
+  browsers: [chromeLauncher()],
+  reporters: [defaultReporter()],
+
+  // TODO Coverage config
+});
```

Comment:
Very basic config. We may want to tweak this in order to try to improve performance, among other things.